gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/DuplicateObservationStatementMapperTest.java

```diff
@@ -263,14 +263,26 @@ public void doesntMergeObservationWhereTheObservationDoesNotEndInEllipses() {
     }
 
     @Test
-    public void removeObservationWhereTheObservationDoesNotEndInEllipsesButCodeableConceptCodesAreTheSame() {
+    public void doNotRemoveObservationWhereTheObservationDoesNotEndInEllipsesButObservationStatementsAllHavePertinentAnnotation() {
         var ehrExtract = createExtract(
             List.of(createObservation("ID-1", "101", "This is an observation doesnt end with ellipses:::"),
                     createObservation("ID-2", "101", "This is an observation which ends with ellipses removed."),
                     generateLinksetComponent("ID-1")));
 
         mapper.mergeOrRemoveDuplicateObservationStatements(ehrExtract);
 
+        assertThat(firstEhrComposition(ehrExtract)).hasSize(3);
+    }
+
+    @Test
+    public void removeObservationWhereTheObservationDoesNotEndInEllipsesButCodeableConceptCodesAreTheSame() {
```

Comment:
Haha, I don't think this summarises the behaviour we want.

```suggestion
    public void removeObservationWithoutPertinentAnnotation() {
```